hasura/metadata/databases/default/tables/public_vaults.yaml

```diff
@@ -32,23 +32,10 @@ select_permissions:
     - decimals
     - deployment_block
     - id
-    - org_id
```

Comment:
should we expose org_id to users who are part of this, so we dont need to change the other queries in this pr?

Reply:
I think this is an information leak. If a user has access to the org row, they can pull the id from the row.